### ISSUE TITLE
Camera opens in its own view, rather inside canvas

### ISSUE THREAD

**Issue by chaitanyakhurana100** [donaldp24/CanvasCameraPlugin#17]
Hi,

I tried different options but got no success in displaying camera view within canvas. Rather it opens in a full screen independent view. Could you please tell what is the possible solution?

Thanks in advance!

**Comment by Ekta09** [comment]
Same Problem with me also. Did you get any solution @chaitanyakhurana100 ? Any help from any one is appreciated. Please Help us.  Thanks in advance.

**Comment by harish118** [comment]
hi,

exactly the same issue here too. Anything in particular I should inspect to check that it opens in a canvas and not in it's own view?

Thanks

**Comment by jeffcrouse** [comment]
same issue here

**Comment by markdegrootnl** [comment]
Yes I have the same problem. But only on Android right? iOS seems to be working as expected.

**Comment by thomascrown** [comment]
+1 for a fix

**Comment by benguela** [comment]
I have the same problem on android

**Comment by bishopZ** [comment]
Same problem on Android 
`<uses-sdk android:minSdkVersion="14" android:targetSdkVersion="22" />`

I tried removing the android platform, setting the minSdk down to 8, then adding the platform back. After getting the SDK version change to fully take, I still got the same camera capture window (rather than the capture in my canvas element).

There are some warnings about this plugin using deprecated Android code. The way that Android deprecates code is often to provide a fallback "compatibility" version of the same thing. I am guessing that the camera capture window we are getting is Android's fallback solution for the deprecated code.  To correct this, someone would need to go into the plugin's code and update whatever was deprecated to use the newer version of the same thing.

One year later, seven people report the same issues, not a single response from the author.

**Comment by sabirali007** [comment]
same issue here

**Comment by nehalchoraria** [comment]
same issue. Anyone got it fixed?
